src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -198,25 +199,9 @@ public boolean isApplicable(Class<? extends AbstractProject> jobType) {
         public ListBoxModel doFillConfigTypeItems() {
             ListBoxModel items = new ListBoxModel();
 
-            // getting the user selected value
-            String temp = getOptionType();
-
-            if (temp.equalsIgnoreCase("ART")) {
-
-                items.add("Average Response Time", "ART");
-                items.add("Median Response Time", "MRT");
-                items.add("Percentile Response Time", "PRT");
-            } else if (temp.equalsIgnoreCase("MRT")) {
-
-                items.add("Median Response Time", "MRT");
-                items.add("Percentile Response Time", "PRT");
-                items.add("Average Response Time", "ART");
-            } else if (temp.equalsIgnoreCase("PRT")) {
-
-                items.add("Percentile Response Time", "PRT");
-                items.add("Average Response Time", "ART");
-                items.add("Median Response Time", "MRT");
-            }
+            items.add("Average Response Time", "ART");
+            items.add("Median Response Time", "MRT");
+            items.add("Percentile Response Time", "PRT");
```

Comment:
The `doFill` methods don't have any parameters. So I honestly don't know how that could be done.